src/sections/block2/Block2.tsx

```diff
@@ -0,0 +1,25 @@
+import s from "./Block2.module.scss"
+
+function Block2 () {
+
+	return (
+		<div className={s.wrapper}>
+			<div className={s.blockTwo}>
+				<div className={s.top}>
+					<h1 className={s.titleTop}>WHO WE ARE</h1>
+				</div>
+				<div className={s.bottom}>
+					<div className={s.titleList}>
+						<h2 className={`${s.titleList__item} ${s.titleList__item_first}`}>SPEED UP DEVELOPMENT</h2>
+						<h2 className={`${s.titleList__item} ${s.titleList__item_second}`}>INCREASE QUALITY</h2>
+						<h2 className={`${s.titleList__item} ${s.titleList__item_third}`}>REDUCE COSTS</h2>
+					</div>
+					<h1 className={s.titleBottom}>WHAT WE DO</h1>
+				</div>
+			</div>
+		</div>
```

Comment:
I change this block in Figma. Because it's not very good when these containers have fullscreen width. Also, I'd made some more changes on [Figma](https://www.figma.com/file/RKAxePr0yVgwoWNQYvBWO7/Untitled?node-id=0%3A1&t=dZ5UMdDpbcOEVoUk-1), check it.